tools/topology/topology2/cavs-sdw-src-gain-mixin.conf

```diff
@@ -167,11 +167,15 @@ Object.Base {
 
         route."2" {
                 source 'gain.2.1'
-                sink 'copier.ALH.2.1'
+                sink 'copier.ALH.2.playback'
```

Comment:
For ALH I am not sure if the dai_index has any meaning. It's my understanding that the same copier or gateway will manage streaming over multiple links.

@bardliao need your help here.